gapic/src/main/com/google/gapid/models/ProfileExperiments.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (C) 2021 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.gapid.models;
+
+import com.google.gapid.proto.service.Service;
+
+public final class ProfileExperiments {
+  public final boolean shouldDisableAnisotropicFiltering;
```

Comment:
Sorry, we're going around in circles here. Boolean variables do not need the `is`, `should`, `was`, etc prefix. Only use those prefixes in function names. This is similar to, for example `int foo` and `int getFoo()`. You wouldn't call the variable `getFoo`. So for example:
```
[public|private] final boolean disableAnisotropicFiltering;

public boolean shouldDisableAnisotropicFiltering {
  return disableAnisotropicFiltering;
}
```

Since, in our case, it's public, we don't need the getter.